### ISSUE TITLE
Unable to have mockserver use own certificate

### ISSUE THREAD

**Issue by cowens5** [jamesdbloom/mockserver#261]
I'm running mock-server on multiple machines, so I want to be able to use my self-signed trusted root certificate -- let's call it hostname -- already present on all of those machines, without having to un/install certs on every machine I will run it on. However, mock-server seems to only be able to use the www.mockserver.com cert. Is there a way for the mock-server to respect a cert that's not mock server's? I have tried changing ConfigurationProperties: javaKeyStoreFilePath, javaKeyStorePassword and sslCertificateDomainName, but that doesn't seem to affect what cert gets validated.

**Comment by james-boswell** [comment]
I have raised a similar issue #263 to this one.

**Comment by bjhartin** [comment]
@cowens5 @james-boswell We've had success doing this simply by copying the mock-server-core/src/main/resources/org/mockserver/socket directory, updating the keystore/cert/key, and then prepending that directory so it's first on the classpath when we run the jar.

**Comment by jamesdbloom** [comment]
I have now added two configuration values to support configuring the private key and X509 certificate for MockServer, these values can be specified using Java properties as follows:

```bash
-Dmockserver.certificateAuthorityPrivateKey=...
-Dmockserver.certificateAuthorityCertificate=...
```

Or using the `ConfigurationProperties` class, as follows:

```java
ConfigurationProperties.certificateAuthorityPrivateKey(String certificateAuthorityPrivateKey)
ConfigurationProperties.certificateAuthorityCertificate(String certificateAuthorityCertificate)
```

This is currently available in the SNAPSHOT version but will be released in the next few weeks.

**Comment by stejskal** [comment]
This addition was very helpful to me, thanks!  i recommend the config docs get updated as well. 
http://www.mock-server.com/mock_server/configuration_properties.html

**Comment by jamesdbloom** [comment]
the documentation issue covers that and will be addressed shortly #115
